### PR TITLE
add docs: troubleshoot high latency

### PR DESCRIPTION
**Pull request by fuyufjh**
## Info

- **Description**

Add the example of barrier latency caused by join amplication.

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -985,9 +985,14 @@ const sidebars = {
         },
         {
           type: "doc",
-          label: "Troubleshoot out-of-memory problems",
+          label: "Troubleshoot out-of-memory",
```
Comment (fuyufjh) [inline sidebars.js:988]: Just to make it more tidy:

Before:
image

After:
image

**Review by WanYixian — APPROVED** [review]
Review done and LGTM now

File: docs/troubleshoot/troubleshoot-high-latency.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+With these tools, you can identify the bottleneck fragments (actors) and the materialized views they belong to. Additionally, refer to the RisingWave Dashboard for the detailed information on materialized views, such as the streaming execution graph or the SQL query.
+
+## Solution
+
+Once you've pinpointed the bottleneck fragment, consider the following actions to resolve the issue:
+
+- Enhance the streaming query performance by removing or rewriting the bottleneck part of the SQL query.
+- Increase the parallelism by adding more nodes into the cluster, or check the SQL query to see if there is any room for optimization.
+
+## Example
```
Comment (fuyufjh) [inline docs/troubleshoot/troubleshoot-high-latency.md]: @WanYixian Please keep the original subtitle: "Example: High latency caused by high join amplification". I plan to add more cases here in the future, and join amplification is one among them.

Comment (WanYixian) [reply docs/troubleshoot/troubleshoot-high-latency.md]: Was trying to keep the subtitles consistent in length, now add it back:)

**Pull request merged**